src/Helper/AttributesTrait.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-view for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-view/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-view/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\View\Helper;
+
+/**
+ * Helper trait for processing HTML tag attributes
+ */
+trait AttributesTrait
+{
+    /**
+     * Converts an associative array to a string of tag attributes.
+     *
+     * @param array $attribs From this array, each key-value pair is
+     * converted to an attribute name and value.
+     *
+     * @return string The XHTML for the attributes.
+     */
+    protected function htmlAttribs($attribs)
+    {
+        $xhtml          = '';
+        $escaper        = $this->getView()->plugin('escapehtml');
```

Comment:
Unfortunately that won't work. There is no method `getView` in this trait.

This example will fail:

```php
class Example
{
	use AttributesTrait;

	public function test()
	{
		$this->htmlAttribs([]);
	}
}
```
This means we have a wrong usage of a trait here.

Reply:
So the fact that the trait is meant to be used by view helpers only and is in the `Laminas\View\Helper` namespace does not help (pun intended)? 🙂 

I will try to think of another approach then. Suggestions are welcome!

Reply:
@froschdesign Maybe I could place these in a class extending `AbstractHelper` and extended by `AbstractHtmlElement`?

Reply:
If we add it as an abstract method to the trait, this should work, however.

Reply:
I was actually just thinking the same and have now done that!